examples/custom-session-validation/README.md

```diff
@@ -0,0 +1,120 @@
+## Feature Example - Invalidate Session Token
+
+This project demonstrates how to invalidate a session when a password is changed using password based authentication in your Keystone system.
+It builds on the [With Auth](../with-auth) example project.
```

Comment:
```suggestion
It builds on the [Authentication](../with-auth) example project.
```

Can we actually rename the `with-auth` example to `authentication`?
That is the banner the README ships with anyway.